daemon/metrics_unix.go

```diff
@@ -28,7 +29,10 @@ func (daemon *Daemon) listenMetricsSock() (string, error) {
 	mux := http.NewServeMux()
 	mux.Handle("/metrics", metrics.Handler())
 	go func() {
-		http.Serve(l, mux)
+		logrus.Debugf("metrics API listening on %s", l.Addr())
```

Comment:
I guess I don't understand why we have two places (here and in cmd/dockerd) where we start the metrics server.

Reply:
This is for metrics plugins.